packages/fast-usdc/test/exos/transaction-feed.test.ts

```diff
@@ -79,6 +79,21 @@ test('happy aggregation', async t => {
   });
 });
 
+test('disabled operator', async t => {
```

Comment:
good; the "unknown transaction" test mixed in testing this, and I wondered about reducing coverage.